packages/store/ts/config/parseStoreConfig.ts

```diff
@@ -220,9 +263,13 @@ const StoreConfigUnrefined = z
 // finally validate global conditions
 export const zStoreConfig = StoreConfigUnrefined.superRefine(validateStoreConfig);
 
-export function parseStoreConfig(config: unknown) {
-  return zStoreConfig.parse(config);
-}
+// Catchall preserves other plguins' options
```

Comment:
```suggestion
// Catchall preserves other plugins' options
```

Reply:
also fixed the same typo in parseWorldConfig